src/ch02/trafficLight.test.ts

```diff
@@ -10,28 +14,49 @@ class Car {
   }
 }
 
-const CYCLE = [TrafficLight.RED, TrafficLight.YELLOW, TrafficLight.GREEN];
+interface TrafficLight {
+  isRed(): boolean;
+  isYellow(): boolean;
+  isGreen(): boolean;
+  updateCar(): boolean;
+}
```

Comment:
```suggestion
interface TrafficLight {
  updateCar(): boolean;
}
```
해당 메서드를 제거하면 어떤 결과가 발생할까요? 제거 유무를 한번 고민 했으면 좋겠습니다.

Reply:
isRed, isYellow, isGreen 을 주석 처리해서 제거해도 코드 상의 오류가 발생하지 않습니다.
Red, Yellow, Green 클래스 내의 isRed, isYellow, isGreen 은 
인터페이스에서 상속 받은 클래스를 구현한게 아니라
개별적으로 클래스를 구현한 꼴이 된 거 같습니다.

Reply:
인터페이스에서 코드상에 오류가 발생하지 않는다면 상속 받고 있는 클래스에서도 더 이상 사용되지 않아 제거가 가능 하지 않을까요? 99쪽에 **필요없는 코드 제거하기** 내용을 읽어보시면 좋을 것 같습니다~